WcaOnRails/spec/factories/users.rb

```diff
@@ -47,7 +47,10 @@
     end
 
     trait :wca_id do
-      wca_id { FactoryGirl.create(:person, name: name).id }
+      transient do
```

Comment:
[This](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#transient-attributes) is just a way to set some variables that can be overridden (see the beginning of the file too, I used a handy way of passing the preferred_events there).